content/2016.09.jedit-trackers-with-kinto-en.md

```diff
@@ -0,0 +1,403 @@
+Title: Getting jEdit Trackers off the Ground using Kinto and Kinto.js
+Date: 2016-09-09
+Lang: en
+Url: jedit-trackers-with-kinto
+Slug: jedit-trackers-with-kinto
+Summary: My experience using Kinto and Kinto.js to browse jEdit tickets offline
+
+# Getting jEdit Trackers off the Ground using Kinto and Kinto.js
+
+This is a guest post by [Eric Le Lay](https://github.com/elelay) on his experience using Kinto.
+
+## Introduction
+
+Do you know [jEdit](http://jedit.org) — *programmer's text editor*?  
```

Comment:
To translate `connaitre`, which to me suggests only slight familiarity, I would probably use something like "Have you heard of". Also, in the project's slogan, I'd probably write "the programmer's text editor" or "a programmer's text editor", but since this is the official project motto (shown on jedit.org), I guess this is fine.